### ISSUE TITLE
Call for help, cannot reproduce the results of learning curves on NL Texas Hold'em trained by DQN

### ISSUE THREAD

**Issue by jiahui-x** [datamllab/rlcard#224]
Hi, I am struggling to get a good training result on NL Texas Hold'em by DQN. I follow your paper to choose hyperparameters as: **the memory size is selected in {2000, 100000}, the discount factor is set to 0.99, Adam optimizer is applied with learning rate 0.00005, and the network structure is MLP with size 10-10 128-128, 512-512 or 512-1024-2048-1024-512**(I have tried them all). But can only get a bad results like:
![s](https://user-images.githubusercontent.com/57308120/122015271-8a3d8780-cdf2-11eb-9728-d86fb6a62cd8.png)
And the result in your paper like:(with the same amount of training timesteps but got much more rewards)
![image](https://user-images.githubusercontent.com/57308120/122015727-f02a0f00-cdf2-11eb-8d6e-69ae56aa3d42.png)

**Comment by daochenzha** [comment]
@jiahui-x Hi, thanks for the feedback. This result in the paper is out-of-date. This is possibly due to multiple factors. First, the environment codebase has some major updates since the first release, particularly the reward has been divided by 2. Second, the current implementation is based on Torch instead of TensorFlow. Thus, your result seems reasonable to me.

The codebase is expected to remain stable in the near future to ensure reproducibility.
